unime/src/lib/components/molecules/ListItemCard.svelte

```diff
@@ -10,6 +10,15 @@
   export let description: string | undefined = undefined;
   export let type: 'data' | 'badge' = 'data';
   export let isTempAsset = false;
+  export let hasBorderRadius = true;
+
+  function addRoundedBorder() {
```

Comment:
But I want to always use the same class, except for in history I don't want to add a certain class. Isn't it better to do it like this? Otherwise everywhere you have to pass rounded-lg class